cli/src/commands/expirationExports.js

```diff
@@ -0,0 +1,53 @@
+import Download from 'lib/models/download';
+import SiteSettings from 'lib/models/siteSettings';
+import { SITE_SETTINGS_ID } from 'lib/constants/siteSettings';
+import { get } from 'lodash';
+import moment from 'moment';
+import { map } from 'bluebird';
+import Organisation from 'lib/models/organisation';
+
+const expireExportsForOrgatinasion = async (organisation) => {
+  const ttl = organisation.settings.EXPIRE_EXPORTS.ttl;
+
+  const todayDate = moment();
+  const cutOffDate = todayDate.subtract(ttl, 'seconds');
+
+  const exports = await Download.find({
+    organisation: organisation._id,
+    upload: { $exists: true },
+    time: { $lt: cutOffDate } // DEBUG ONLY, uncomment
```

Comment:
So, I'm thinking we refactor this a touch so that the TTL is used to set an `expiresAt` date on each model (rather than a global expires TTL); this method would just find all downloads where `expiresAt: {$lt: new Date() }` (aka now)

My fear is that someone would drop the TTL for their org to 60s (or something silly like that) by accident, and then the scheduler may come along and dutifully delete all of their valuable data which was made more than a minute ago.

Having the expiry date set on the export means that the TTL is calculated at the time the export is made, and has a fixed expiry (or maybe has no expiry at all). An added bonus to that will be we never have to "calculate" the expiry in the UI either - it either has one or doesn't